.gitignore

```diff
@@ -25,6 +25,8 @@
 engine/.clang_complete
 /.vscode
 .npm
+/**/.project
+/**/.gradle
```

Comment:
While I didn't mean to check this in, I wonder if we actually are supposed to have these checked in?
Should I keep this change?

Reply:
don't we use .project for test in game.project ?

Reply:
Not sure what you mean?
This isn't "*.project", but ".project", which seems like it's written by vscode or intellij :/